client/app/cloud/project/add/cloud-project-add.html

```diff
@@ -39,7 +74,7 @@ <h1 data-ng-if="CloudProjectAddCtrl.data.projectsCount > 0" data-translate="cpa_
                 </div>
 
                 <!-- contracts: okey (only shown if its the first project) -->
-                <div class="row" data-ng-if="!CloudProjectAddCtrl.data.agreements.length && CloudProjectAddCtrl.data.projectsCount === 0">
+                <div class="row" data-ng-if="!CloudProjectAddCtrl.data.agreements.length && CloudProjectAddCtrl.data.isFirstProjectCreation">
```

Comment:
do we really need agreement, there is no agreement in express order, firstProjectCreation can be enough ?

Reply:
If the user is creating his first project and that for some reasons there are no agreements to display, we have to hide the agreements section.